test/general.test.js

```diff
@@ -399,6 +399,14 @@ describe('Checker', function () {
 });
 
 describe('format', function () {
+    beforeEach(function () {
```

Comment:
Would be great to have a separate test case that shows that files are being removed when errors are present.

Reply:
Yes, that would be better, easier to maintain!